src/globus_sdk/config/environments.py

```diff
@@ -50,8 +52,21 @@ def get_by_name(cls, envname: str) -> Optional[Type["EnvConfig"]]:
         return cls._registry.get(envname)
 
 
-def get_service_url(environment: str, service: str) -> str:
+def get_service_url(service: str, environment: Optional[str] = None) -> str:
+    """
+    Return the base URL for the given service in this environment. For example:
+
+    >>> from globus_sdk.config import get_service_url
+    >>> get_service_url("auth", environment="sandbox")
+    'https://auth.sandbox.globuscs.info/'
+    >>> get_service_url("search", environment="test")
+    'https://search.api.test.globuscs.info/'
```

Comment:
```suggestion
    >>> get_service_url("auth", environment="preview")
    'https://auth.preview.globus.org/'
    >>> get_service_url("search", environment="production")
    'https://search.api.globus.org/'
```

Because only `preview` and `production` are "public" (exactly what this means is subject to debate) and this is going into docs, I think we should avoid using other environments here.

Reply:
Good point, thanks. I'm just going to apply these suggestions directly.